computer.js

```diff
@@ -0,0 +1,124 @@
+var move = [];
+var numDoughnutsInTower = [NUM_DOUGHNUT, 0, 0];
+var cnt = 0;
+var HEIGHT = 250;
+
+
+function computerDet(doughnut, below, end)
+{
+	this.doughnut = doughnut;
+	this.below = below;
+	this.end = end;
+}
+
+
+function instructions(N, start, aux, end)
+{
+	if(N>=0)
```

Comment:
Spacing: One space after `if`, spaces around operators such as `>=`. Here and everywhere below.